Group1 Assignment

isha123456 opened the pull request:
Group1 Assignment

isha123456 committed:
Group1 Assignment

vagdur committed:
overwrite old L12 file instead

Doing it like this means git automatically shows a diff of what you have changed, instead of making me search for it manually

vagdur commented:
[lect12_xca_group1.pdf](https://github.com/vagdur/Grafteori-1MA170/files/13725995/lect12_xca_group1.pdf)
[lect12_group1.pdf](https://github.com/vagdur/Grafteori-1MA170/files/13725996/lect12_group1.pdf)

isha123456 committed:
Group1 assignement: updated after oral feedback

vagdur committed:
correct names of participants

As I understood it, Gholam withdrew from the group, so removing his name from the exercise

(Also removed the group number to make it all fit on one line)

vagdur committed:
add forgotten word

vagdur committed:
remove editor names from main lecture file

you get credits in the pull request history and in the exercises, when the thing is merged

vagdur committed:
fix locations of graphics files

they should all be sorted into their respective folders - so they've all been moved now, and the TeX source updated to point to the right files

vagdur committed:
tweak figure and remove unnec subscr

Two changes:

1. There's no reason this needs a subscripted 1, when there are no other graphs in play - so I changed it to just a G.
2. I tweaked the figure by cropping it to its actual content (removing that red dot as well) and making its background uniformly transparent. This also allowed it to be scaled better by TeX - TeX doesn't see the margins in the image file itself when giving it a margin, so preferably figures should have zero margins in the image file, so TeX can assign the right margins itself.

vagdur committed:
crop other new figures

cf. previous commit note as to why

vagdur committed:
slightly shrink figure display

it doesn't need to be quite so large to be legible

vagdur committed:
move end of Lemma block

this does absolutely nothing for the actual look of the document - but it does work slightly more nicely in the editor, with how you can collapse entire blocks - so I like to keep the Proof block contained in the Lemma block

vagdur committed:
|E| -> \abs{E}, and fix resulting errors

Not sure what was going on there to make \abs break, but it should work now - something about the order of includes and begin{document}s?

vagdur committed:
TeX indentation fixes

vagdur committed:
remove difficult alternative solution

I am still not confident that this is actually *wrong* - but it is definitely still confusing, so it seems counterproductive to provide this exercise solution to the other students. Thus I am removing it from the version getting merged into the main repository.

vagdur committed:
fix spacing and linebreaks

Using \\ is almost always bad practice outside of an equation. Using line breaks in the TeX that do not change the appearance of the document is *always* bad practice, and has led to actual errors (albeit minor) for me in the past in the mathematical content of what I was writing.

Using \noindent should be done very sparingly - it looks noticeably odd to have different indentation than standard TeX, so it should be done for a reason.

vagdur committed:
typo fixes

vagdur committed:
punctuation

vagdur committed:
Moved source to footnote

The % in the URL was messing things up, so I used an URL shortener for that

vagdur committed:
language tweaks and TeX fixes

The main comment here is that *whenever you write a maths symbol, it must be in an equation* - TeX uses visibly different fonts for G and $G$, so whenever you are talking about a graph named G in TeX, you write its name as $G$, to get the right font for it. Not doing so looks unprofessional.

vagdur committed:
add exercise solutions to readme
